codespell_lib/_codespell.py

```diff
@@ -722,8 +723,6 @@ def main(*args):
                     fname = os.path.join(root, file_)
                     if glob_match.match(fname):  # skip paths
                         continue
-                    if not os.path.isfile(fname) or not os.path.getsize(fname):
-                        continue
```

Comment:
We might as well handle this in parse_file so we can deal with all the filename checking in one place.